### ISSUE TITLE
Mobile Design Cleanup

### ISSUE THREAD

**Issue by Auroratide** [Auroratide/poke5e#111]
Stacking numbers in search bar:
<img width="129" alt="stacking numbers in search bar" src="https://github.com/user-attachments/assets/69488fb1-16db-4fe5-a3f0-4399434e0e05" />

Too many columns causing overflows:
<img width="372" alt="four columns of text in tables" src="https://github.com/user-attachments/assets/ee93a322-8ae4-4447-9b6b-faa21f9478ee" />

Footer is very large, eating up precious screen space:
<img width="388" alt="big footer" src="https://github.com/user-attachments/assets/cca4df7c-9ae5-44ed-8813-b1747502b752" />

Attributes table gets squished:
<img width="339" alt="all the numbers squish together" src="https://github.com/user-attachments/assets/73dabdc1-93f4-4442-9358-371af9959b34" />

Too much padding on lists:
<img width="338" alt="list in card is padded so much to the right" src="https://github.com/user-attachments/assets/ef50145b-33a3-4018-abf0-99b4d6ea5bd6" />

Large tables overflow the card:
<img width="343" alt="table falls off the card" src="https://github.com/user-attachments/assets/33d1d4d2-3d5c-447f-bd16-2a750af13ca5" />

Lack of padding on the search bar looks sloppy:
<img width="377" alt="edges touch the screen edges" src="https://github.com/user-attachments/assets/4b6d683e-108d-4001-a7fb-29483be49a56" />

Input fields fall off the card view:
<img width="342" alt="input field is too long" src="https://github.com/user-attachments/assets/02b47531-84e7-4e43-b0bd-17a1bcfb1873" />
<img width="364" alt="more off screen" src="https://github.com/user-attachments/assets/1dc7af98-5bce-4c8a-bfc5-08aa2f11adb4" />

The amount of space for pokemon only allows me to view one at a time:
<img width="380" alt="pokemon on trainer screen are squished" src="https://github.com/user-attachments/assets/2d8d3748-5f7d-4642-9033-2690d9aca7cc" />

This spacing just feels wrong:
<img width="348" alt="avatar editor spacing" src="https://github.com/user-attachments/assets/e5e60f57-57da-49cd-9cda-187e47e4eb94" />

**Comment by Auroratide** [comment]
- [ ] Numbers in search bar
- [ ] Too many columns
- [ ] Footer too big
- [ ] Attributes Table
- [ ] List Padding
- [ ] Table Overflow
- [ ] Search Bar Padding
- [ ] Input Lengths
- [ ] Different way to think about lists
- [ ] Avatar Input Spacing
